server-kube/src/main/java/work/thomas/serverkube/EmployeeController.java

```diff
@@ -56,10 +60,7 @@ public Employees namesGET() {
     @ResponseBody
     public void namePOST(@RequestBody final String product)
```

Comment:
product seems an odd name for the param?

Reply:
Yeah, that was a leftover from some example code I copied. Fixed now.